i18n/translation-ko.json

```diff
@@ -289,7 +289,7 @@
       "toolbarGroup-Annotate": "주석",
       "toolbarGroup-Shapes": "도형",
       "toolbarGroup-Insert": "삽입",
-      "toolbarGroup-Measure": "측정",
+      "toolbarGroup-Measure": "단위",
```

Comment:
Based on its context we think it should be:
```suggestion
      "toolbarGroup-Measure”: “측정",
```

![image](https://user-images.githubusercontent.com/10825527/175136866-43f16afb-109a-46c4-8411-fa227ccc6634.png)